src/templates/static/metatrader/download.jsx

```diff
@@ -1,108 +1,228 @@
 import React from 'react';
 import SeparatorLine from '../../_common/components/separator_line.jsx';
 
-const DownloadBlock = ({
-    badge,
-    desc,
-    download,
-    href,
-    image,
-    os,
-    target,
-    title,
-}) => (
-    <div className='gr-4 gr-6-m gr-padding-10'>
-        <img className='gr-6 gr-centered' src={it.url_for(`images/pages/metatrader/icons/${image}.svg`)} />
+const DownloadBlock = (
+    {
+        badge,
+        desc,
+        download,
+        href,
+        image,
+        os,
+        target,
+        title,
+        children,
+    }
+) => (
```

Comment:
if you put the curly brackets on the same line as the brackets and indent everything back once, it will be the same as it used to be and consistent with our other files

Reply:
same for the blocks below